File: ios/AppObjC/AppObjC/ViewController.m
//
//  ViewController.m
//  AppObjC
//
//  Created by Levent Oral on 24.12.2020.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    MobilerDevSDK *mobilerDevSDK = [[MobilerDevSDK alloc] init];
    NSLog(@"%@", [mobilerDevSDK testMethod]);
}

@end

File: ios/App/App/ViewController.swift
//
//  ViewController.swift
//  App
//
//  Created by Levent Oral on 24.12.2020.
//

import UIKit
import MobilerDevSDK

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        let mobilerDevSDK = MobilerDevSDK()
        print(mobilerDevSDK.testMethod())
    }
}

